interface/resources/qml/controls/+webengine/FlickableWebViewCore.qml

```diff
@@ -98,6 +99,7 @@ Item {
 
         width: parent.width
         height: parent.height
+        backgroundColor: (flick.useBackground) ? "white" : "transparent"
```

Comment:
I tried using {} per standard QML conventions, but apparently that threw errors saying something along the lines of "Cannot convert QJSValue to QColor."

However, it appears from the docs https://doc.qt.io/qt-5/qml-qtwebengine-webengineview.html#backgroundColor-prop that the default value was white anyway. So I think it would have ended up being that way anyway? I sincerely hope so.

Reply:
Actually, upon testing it does work, but it does throw that warning/error still.

Reply:
According to this beautiful test page
`<!DOCTYPE html>
<html>
<head>
<style>
body {
  background-color: yellow;
}
</style>
</head>
<body>

<h1 style="font-size: 64px; color: white;">This is a test. (:</h1>

</body>
</html>`
Setting it to "white" does not affect the background if there is one already on the body for example. I think the web as a whole defaults to white regardless if you don't specify, so it will never seem out of the ordinary, I imagine.